File: classe3/src/mtd_construtor/Automovel.java
package mtd_construtor;

public class Automovel {
	private int ano;
	String modelo;
	String marca;
	private static String cor;
	double preco;
	
	Automovel(){}
	
	Automovel(int ano, String marca, String modelo, double preco){
		this.ano= ano;
		this.modelo = modelo;
		this.marca = marca;
		this.preco = preco;		
	}
	
	static {
		cor = "verde";
	}
  public void mostracarro()
  {
	  System.out.println("\n carro marca:\t" + marca + "\n modelo:\t"+ modelo + "\n ano:\t" + ano + "\n cor:\t" + cor + "\n preco:\t" + preco);
  }
}

File: comparacao_python/mtd_construtor/Automovel.py
class Automovel:
    __cor = "verde"

    def __init__(self, ano=0, modelo="", marca="", preco=0):
        self.__ano = ano
        self.modelo = modelo
        self.marca = marca
        self.preco = float(preco)

    
    def mostrar_carro(self):
        print(f"\nano: {self.__ano}\t"
              f"\nmodelo: {self.modelo}\t"
              f"\nmarca: {self.marca}\t"
              f"\ncor: {Automovel.__cor}\t"  
              f"\npreco:{self.preco}\t")


